base_ubl/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# © 2016-2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Base UBL',
+    'version': '12.0.1.0.0',
+    'category': 'Hidden',
+    'license': 'AGPL-3',
+    'summary': 'Base module for Universal Business Language (UBL)',
+    'author': 'Akretion,Odoo Community Association (OCA)',
+    'website': 'https://github.com/oca/edi/',
+    'depends': [
+        'product_uom_unece',
```

Comment:
Now called `uom_unece`

Is it community-data-files already in oca_dependencies.txt?

Reply:
yes, it is.